bluebrain/repo-bluebrain/packages/py-multiscale-run/package.py

```diff
@@ -28,7 +28,10 @@ class PyMultiscaleRun(PythonPackage):
 
     depends_on("gmsh@4:", type=("build", "run"))
     depends_on("julia@1.6:", type=("build", "run"))
-    depends_on("neurodamus-neocortex+ngv+metabolism~~coreneuron@1.15:", type=("build", "run"))
+    depends_on(
+        "neurodamus-models model=neocortex +ngv+metabolism~~coreneuron@1.15:",
```

Comment:
@cattabiani, @tristan0x just so that you are aware. This is the new neurodamus package that should be used.